src/cc2olx/tools/README.rst

```diff
@@ -16,8 +16,10 @@ The tool also optionally takes as a keyword argument an ``--output-csv``.
 
 The ``course-id`` argument is the ID of the course as it appears in Studio. For example, ``course-v1:edX+111222+111222``.
 
-The ``directory`` argument is a directory containing video files that will be uploaded to edX's video encoding pipeline.
+The ``directory`` argument is a directory (such as the one created by the video download tool) containing video files and transcripts, that will be uploaded to edX's video encoding pipeline.
```

Comment:
How should this directory be structured? Will all of the video files and transcripts be within the same directory?

Reply:
I added some clarification to it's file searching behavior in this section. Does that help on this point?